Api/Src/Models/Course.ts

```diff
@@ -1,9 +1,22 @@
 //Wspolne dla wszystkich uczestnikow i mentorow
-
+import * as mongoose from 'mongoose';
 import {Section} from './Section'
-interface Course {
-    id: number,
+
+export interface Course {
     name: string,
     sections: Section[],
     description?:string
-}
\ No newline at end of file
+}
+
+const CourseSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        index: true
+    },
+    description: {
+        type: String
+    }
+  }, {timestamps: true});
```

Comment:
@JustynaSmoliga isn't there "sections" missing? I'm trying to send a post request and section cannot be created on my end (but maybe it's me doing something wrong :) )

Reply:
Endpoints for sections missing, but I think, that we should do this in new task. Issue for sections task was added by Robert.

Reply:
all right, in this case this one is resolved :)